package.json

```diff
@@ -27,8 +27,8 @@
   "dependencies": {
     "@decentral.ee/web3-helpers": "^0.5.3",
     "@slack/webhook": "^6.1.0",
-    "@superfluid-finance/ethereum-contracts": "1.9.1-dev.083b723.0",
-    "@superfluid-finance/metadata": "^1.1.27",
+    "@superfluid-finance/ethereum-contracts": "1.9.0",
+    "@superfluid-finance/metadata": "1.1.27",
```

Comment:
is there a reason for this change?

Reply:
I am unable to build on local with that dependency - building of docker image fails, as well as running docker compose config.

Reply:
I mean the change from `^1.1.27` to `1.1.27` for `@superfluid-finance/metadata`